test/plugin/constants.rb

```diff
@@ -560,10 +564,10 @@ module Constants
     # Non-utf8 string tags.
     "nonutf8#{[0x92].pack('C*')}" => 'nonutf8%20',
     "abc#{[0x92].pack('C*')}" => 'abc%20',
-    "#{[0x92].pack('C*')}" => '%20',
+    [0x92].pack('C*').to_s => '%20',
```

Comment:
[Optional] Unrelated to this PR, but `Array#pack` seems to already produce a `String`:
```
2.2.4 :001 > [0x92].pack('C*').class
 => String 
```

Reply:
Changed.